spec/lib/data_update_scripts/index_reading_list_reactions_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+require Rails.root.join("lib/data_update_scripts/20200415200651_index_reading_list_reactions.rb")
+
+describe DataUpdateScripts::IndexReadingListReactions, elasticsearch: true do
+  it "indexes feed content(articles, comments, podcast episodes) to Elasticsearch" do
+    reactions = Array.new(3).map { create(:reaction, category: "readinglist") }
```

Comment:
shortcut idea for the future: `reactions = create_list(:reaction, 3, category: "readinglist")`

Reply:
I knew there was a shortcut but I couldn't remember the syntax! Going to change it so I can get in the habit, thanks!